_posts/2020-04-07-trying-out-julia.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: Trying out Julia programming language
+author: hnybom
+excerpt: >
+  This post is about the Julia programming language and my journey starting from zero knowledge to, well, some knowledge of the language. So if you, like me, are interested trying out Julia language keep on reading. If you are already familiar with the subject you can safely skip this one.
+tags:
+ - Julia
+---
+
+## Let's try out Julia
+
+Few forewords. I have never tried Julia programming language before but I've read couple of headlines about it. The main motivations to me are
+
+* Julia seems to be gaining some momentum in the programming and especially data science community.
+* Even as Julia is dynamically typed it has some static typing going on by enabling the coder to declare a type if he or she wants to. This is akin of Python's type annotation though it seems a little but more powerful. There is a long documentation about this at [docs.julialang.org/en/v1/manual/types/](https://docs.julialang.org/en/v1/manual/types/). One very positive thing I read there is that you can only sub type abstract types not concrete ones, as a Spring framework user this warms my mind a lot ;) !
+* The benchmarks seems to indicate it's much faster than the main competition Python. This is due the fact the language is compiled to native code using LLVM.
+* You can utilize Python packages.
+* I have some background with MatLab and similar plotting stuff seems to be the forte of Julia.
+
+Plotting is actually the "entry point" for this blog. I'll try to do same kind of stuff I did with MatLab with Julia. So basically this means scripting some plotting. So I won't delve into the finer points of e.g. building server side software.
```

Comment:
This blog deals about plotting graphs in Julia (??)

the same kind of stuff

So basically ehkä pois kokonaan. Ja varsinkin vielä seuraavan lauseen So alku. This means... and I won't delve... esim.

finer points vaikuttaa finglishmiltä. Deeper aspects?

Reply:
https://idioms.thefreedictionary.com/finer+points ihan validi termi